pkg/deploy/identity-provider/identity_provider.go

```diff
@@ -75,6 +76,7 @@ func SyncIdentityProviderToCluster(deployContext *deploy.DeployContext) (bool, e
 		return false, err
 	}
 	keycloakURL := protocol + "://" + endpoint
+	deployContext.InternalService.KeycloakHost = fmt.Sprintf("%s://%s.%s.svc.cluster.local:%d", "http", "keycloak", deployContext.CheCluster.Namespace, 8080)
```

Comment:
I guess instead of `cluster.local` we have to use `spec.server.serviceHostnameSuffix`

Reply:
Yes, I will do it after merge https://github.com/eclipse/che-operator/pull/468

Reply:
I used `serviceName.namespace.svc`